client/app/main/main.scss

```diff
@@ -79,18 +79,40 @@
 }
 
 .community-image {
-    width: 90%;
+    width: 180px;
     float: left;
-    margin-right: $base-padding;
+    margin-right: 20px;
```

Comment:
Padding should never be pixels (for responsiveness), and this should be stored as a variable. `$`